gms/client/src/main/java/com/linkedin/dataprocess/client/DataProcesses.java

```diff
@@ -93,19 +101,34 @@ private DataProcessUrn getUrnFromKey(@Nonnull ComplexResourceKey<DataProcessKey,
   @Override
   public CollectionResponse<DataProcess> search(@Nonnull String input, @Nullable Map<String, String> requestFilters,
       @Nullable SortCriterion sortCriterion, int start, int count) throws RemoteInvocationException {
-    throw new UnsupportedOperationException(
-        String.format("%s doesn't support search feature yet,",
-            this.getClass().getName())
-    );
+    final Filter filter = (requestFilters != null) ? newFilter(requestFilters) : null;
+    DataProcessesFindBySearchRequestBuilder requestBuilder = DATA_PROCESSES_REQUEST_BUILDERS
+        .findBySearch()
+        .inputParam(input)
+        .filterParam(filter)
```

Comment:
you can simply use `newFilter(requestFilters)` here and remove line 104. I will cleanup the code for other clients as well.